src/main/java/com/amazon/opendistroforelasticsearch/sql/query/AggregationQueryAction.java

```diff
@@ -247,6 +245,12 @@ private AggregationBuilder getGroupAgg(Field field, Select select) throws SqlPar
         }
 
         if (!refrence) {
+            for (Field f: select.getFields()) {
+                if (f.getAlias()!=null && f.getAlias().equals(field.getName())) {
```

Comment:
Could we save the null check by switch 2 sides like `field.getName().equals(f.getAlias())`. I assume field name is required?

Btw, nitpick about the space. I think we need to add this rule to our checkstyle config. :)

Reply:
Changed entire method to be more readable